inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/debezium/table/RowDataDebeziumDeserializeSchema.java

```diff
@@ -797,7 +797,32 @@ private Object getValueWithSchema(Object fieldValue, String schemaName) {
 
     @Override
     public void deserialize(SourceRecord record, Collector<RowData> out) throws Exception {
-        deserialize(record, out);
+        Envelope.Operation op = Envelope.operationFor(record);
```

Comment:
There are same code in public void deserialize(SourceRecord record, Collector<RowData> out,TableChange tableSchema) , maybe reuse them is a better idea

Reply:
Thanks, it has done.